guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -5,18 +5,12 @@ include::common/header.adoc[]
 
 = {UpgradingDocTitle}
 
-In this guide, the terms upgrade, update, and migrate have the following meanings:
+In this guide, the terms upgrade, and migrate have the following meanings:
 
 Upgrading::
 The process of advancing your {ProjectServer} and {SmartProxyServer} installations from a y-stream release to the next, for example {Project} {ProjectVersionPrevious} to {Project} {ProjectVersion}.
 For more information, see xref:upgrading_overview[].
 
-Updating::
-The process of advancing your {ProjectServer} and {SmartProxyServer} installations from a z-stream release to the next, for example {Project} {ProjectVersion}.0 to {Project} {ProjectVersion}.1.
-ifdef::satellite[]
-For more information, see xref:updating_{project-context}[].
-endif::[]
-
 Migrating::
 The process of moving an existing {Project} installation to a new instance.
```

Comment:
I would move the Upgrading definition to `upgrading_overview.adoc` and remove this whole thing. We don't have any migration in this guide anyway, do we?

Reply:
Another option would be including all three definitions in the glossary of terms, which is included within the Planning guide. Perhaps this option is more sane. :)

Reply:
Migration can still be a part of Project :smile: 
The reason is, after some interval in versions, there is the possibility of dual OS version support. At this stage, end-users can migrate the whole project to a newer OS version. For eg., EL 7 and EL 8 support in `3.1`

Reply:
Added all of them to Glossary :+1: